site/docs/hosting/deno-deploy.md

```diff
@@ -15,27 +15,31 @@ It's just like the other many serverless platforms, but dedicated for Deno apps.
 
 ```ts
 import { webhookCallback } from "https://deno.land/x/grammy/mod.ts";
+import { serve } from "https://deno.land/x/sift@0.5.0/mod.ts";
 // You might modify this to the correct way to import your `Bot` object.
 import bot from "./bot.ts";
 
-import { serve } from "https://deno.land/std/http/server.ts";
-
 const handleUpdate = webhookCallback(bot, "std/http");
 
-serve(async (req) => {
-  if (req.method == "POST") {
-    try {
-      return await handleUpdate(req);
-    } catch (err) {
-      console.error(err);
-      return new Response();
+serve({
+  ["/" + Deno.env.get("TOKEN")]: async (req) => {
+    if (req.method == "POST") {
+      try {
+        return await handleUpdate(req);
+      } catch (err) {
+        console.error(err);
+      }
     }
-  }
-
-  return new Response();
+    return new Response();
+  },
+  "/": () => {
+    return new Response("Hello world!");
+  },
 });
 ```
 
+We advise you to have your handler on some secret path rather than root. Here, we are using the bot token.
```

Comment:
I recommand that we could add the example for it.

```suggestion
We advise you to have your handler on some secret path rather than root(`/`). Here, we are using the bot token(`/<bot token>`).
```

Reply:
I agree.